web_responsive/static/src/legacy/scss/web_responsive.scss

```diff
@@ -343,71 +255,26 @@ html .o_web_client .o_action_manager .o_action {
             display: none;
         }
     }
-
-    // Sided chatter, if user wants
-    .o_chatter_position_sided & {
-        @include media-breakpoint-up(lg) {
-            .o_form_view:not(.o_form_nosheet) {
-                display: flex;
-                flex-flow: row nowrap;
-                height: 100%;
-
-                .o_form_sheet_bg {
-                    flex: 1 1 auto;
-                    overflow: auto;
-
-                    > .o_form_sheet {
-                        min-width: unset;
-                    }
-                }
-
-                .o_FormRenderer_chatterContainer {
-                    border-left: 1px solid gray("400");
-                    flex: 0 0 $chatter_zone_width;
-                    max-width: initial;
-                    min-width: initial;
-                    overflow: auto;
-
-                    .o_chatter_header_container {
-                        padding-top: $grid-gutter-width * 0.5;
-                        top: 0;
-                        position: sticky;
-                        background-color: $o-view-background-color;
-                        z-index: 1;
-
-                        // Scrollable input text to avoid hide conversation & buttons
-                        .o_composer_text_field {
-                            max-height: 120px;
-                            overflow-y: auto !important; /* Forced because Odoo uses inline style */
-                        }
-                        .o_attachments_list {
-                            overflow: auto;
-                            max-height: $o-mail-attachment-image-size * 3;
-                            margin-top: 0.4em;
-                        }
-                        .o_attachments_previews {
-                            overflow: auto;
-                            max-height: $o-mail-attachment-image-size * 6;
-                        }
-                    }
-                }
-            }
-        }
-    }
 }
 
 // Sticky Header & Footer in List View
-.o_list_view {
-    .table-responsive {
+.o_list_renderer {
+    &.table-responsive {
```

Comment:
Last commit here case extra gap:
![extra-gap](https://user-images.githubusercontent.com/4132418/222058592-1df82ceb-d340-4c31-bcbc-47c8f015ea45.png)

Reply:
Yes , reverted the changes, thanks